config.yml

```diff
@@ -18,6 +18,8 @@ parameters:
     # for learner dashboard
     use_only_custom_elapsed_time: true
     display_activity_elapsed_time: false
+    # test mode for extra-actity
+    extra_activity_mode: false
```

Comment:
je serais pour activer la fonctionnalité par défaut, et mettre le commentaire "'if true, the extra activities can be added to the courses''

Reply:
C'était pour les tests, donc je le laisse à false.
et quand tout sera validé sans trop de changements, je le passe à true pour éviter que les changements apparaissent chez tout le monde.

Reply:
peut etre decrire cela comme une feature, et donc dans le fichier de conf, on peut décider d'utiliser la feature ou pas (comme avec les tab_in_activities).
Et on peut discuter des features présentes par défaut plus tard?

Reply:
Totalement d'accord pour une discussion plus tard des features par défaut en considérant pour le moment qu'extra activity n'est pas par défaut (c'est une nouveauté)

Reply:
oui c'était juste une remarque, à discuter plus tard ce qu'on décide comme feature présente.